### PR TITLE
Set cwd for workers and remove --block

### PR DESCRIPTION
**Pull request by ehsantn**
Sets current working directory for engines to avoid path errors for files, and sets engine blocking globally to remove `--block`. Also, the data is move to a directory.

### CODE REVIEW

File: bodo_tutorial.ipynb [hunk 1]
```diff
@@ -98,17 +98,33 @@
...
+      "1  2\n",
+      "2  3\n",
+      "3  4\n",
+      "4  5\n",
+      "5  6\n",
+      "6  7\n",
+      "7  8\n"
+     ]
+    }
+   ],
```
Comment (juanjgalvez) [inline bodo_tutorial.ipynb:119]: Can you clear this output?

File: bodo_tutorial.ipynb [hunk 2]
```diff
@@ -177,26 +193,41 @@
...
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[stdout:0] /Users/ehsan/dev/Bodo-tutorial\n",
+      "[stdout:1] /Users/ehsan/dev/Bodo-tutorial\n",
+      "[stdout:2] /Users/ehsan/dev/Bodo-tutorial\n",
+      "[stdout:3] /Users/ehsan/dev/Bodo-tutorial\n"
+     ]
+    }
+   ],
```
Comment (juanjgalvez) [inline bodo_tutorial.ipynb:209]: Maybe we should clear this output too, no point in showing it in documentation.

File: bodo_tutorial.ipynb [hunk 3]
```diff
@@ -1532,11 +1561,13 @@
     {
      "data": {
       "text/plain": [
-       "['ipython-input-10825d883f44.mean_power_speed-4444615264.py38.1.nbc',\n",
-       " 'ipython-input-10825d883f44.mean_power_speed-4444615264.py38.nbi']"
+       "['ipython-input-d798fc88f40a.f-540f173005.py38.1.nbc',\n",
+       " 'ipython-input-6495a8443ea5.mean_power_speed-4444615264.py38.1.nbc',\n",
+       " 'ipython-input-d798fc88f40a.f-540f173005.py38.nbi',\n",
+       " 'ipython-input-6495a8443ea5.mean_power_speed-4444615264.py38.nbi']"
```
Comment (juanjgalvez) [inline bodo_tutorial.ipynb:1567]: This output is confusing I think (for documentation). Shows two different sets of cache files (probably from running the cell with two different source versions).

Comment (juanjgalvez) [reply bodo_tutorial.ipynb:1567]: Or different notebook executions. Just clear the cache and re-run the cell.

File: bodo_tutorial.ipynb (hunk 1 above)
Comment (ehsantn) [reply bodo_tutorial.ipynb]: Done

File: bodo_tutorial.ipynb (hunk 3 above)
Comment (ehsantn) [reply bodo_tutorial.ipynb]: Done

File: bodo_tutorial.ipynb (hunk 2 above)
Comment (ehsantn) [reply bodo_tutorial.ipynb]: Done

**Pull request merged**